bibliography.bib

```diff
@@ -463,21 +463,19 @@ @article{brown_identification_2016
 	file = {ScienceDirect Full Text PDF:/home/teddy/Zotero/storage/9H5GZ7QA/Brown et al. - 2016 - Identification of fuel cycle simulator functionali.pdf:application/pdf;ScienceDirect Snapshot:/home/teddy/Zotero/storage/Z6WQH4Q4/S0306454916303383.html:text/html;ScienceDirect Snapshot:/home/teddy/Zotero/storage/NFQC7CXG/S0306454916303383.html:text/html}
 }
 
-@article{bae_arfc/transition-scenarios:_2018,
-	series = {{GitHub}},
-	title = {arfc/transition-scenarios: {Synergistic} {Spent} {Nuclear} {Fuel} {Dynamics} {Within} the {European} {Union} v2.0.0},
+
+@misc{bae_arfc/transition-scenarios:_2018,
+	title = {arfc/transition-scenarios: {Synergistic} {Spent} {Nuclear} {Fuel} {Dynamics} {Within} the {European} {Union}},
 	shorttitle = {arfc/transition-scenarios},
-	url = {https://doi.org/10.5281/zenodo.1210302},
-	doi = {10.5281/zenodo.1210302},
+	url = {https://zenodo.org/record/1477459},
 	abstract = {This release contains code to reproduce the plots used in the paper Synergistic Spent Nuclear Fuel Dynamics Within the European Union by Jin Whan Bae, Clifford Singer, Kathryn Huff},
-	urldate = {2018-04-03},
-	journal = {Zenodo},
-	author = {Bae, Jin Whan and Park, Gyu Tae and Huff, Katy and Chee, Gwendolyn},
-	month = mar,
+	urldate = {2018-11-03},
+	publisher = {Zenodo},
+	author = {Bae, Jin Whan and Chee, Gwendolyn and Huff and Kennelly, Tyler and PEP8 Speaks and Wilson, Paul},
```

Comment:
I seem to be lacking a first name in this Zenodo DOI?

Reply:
fixed it! sorry about that